### PR TITLE
[#754] Remove logging from ListParts

### LINKED ISSUE #754: Add `CID` and `OID` to logs
It would be nice to see in logs not only object name but its oid also. It can help debug issues quicker.
For bucket operations (e.g. `HeadBucket`) it would be nice to log CID. (Currently we just return it in response header)

### PR DESCRIPTION
**Pull request by ir0nbee**
Closes #754 

Removed logging from ListParts at `layer/multipart_upload.go` since more detailed logging happens in `getUploadParts` that gets called in ListParts.

Signed-off-by: Artem Tataurov <a.tataurov@yadro.com>

### CODE REVIEW

**Comment by KirillovDenis** [comment]
It seems we can also remove 
https://github.com/nspcc-dev/neofs-s3-gw/blob/2b905283f65a330714e38955b699a1d2abeca6bb/api/layer/multipart_upload.go#L546-L550
for the same reason

**Comment by KirillovDenis** [comment]
And if we start drop redundant logging then let's drop the following:
* [this](https://github.com/nspcc-dev/neofs-s3-gw/blob/2b905283f65a330714e38955b699a1d2abeca6bb/api/handler/multipart_upload.go#L351-L356) because of [that](https://github.com/nspcc-dev/neofs-s3-gw/blob/2b905283f65a330714e38955b699a1d2abeca6bb/api/layer/multipart_upload.go#L233-L237)
* [this](https://github.com/nspcc-dev/neofs-s3-gw/blob/2b905283f65a330714e38955b699a1d2abeca6bb/api/handler/put.go#L733) because of [that](https://github.com/nspcc-dev/neofs-s3-gw/blob/2b905283f65a330714e38955b699a1d2abeca6bb/api/handler/put.go#L748) (or we can merge them into one log with `info` level right after bucket was created) 
* [this](https://github.com/nspcc-dev/neofs-s3-gw/blob/2b905283f65a330714e38955b699a1d2abeca6bb/api/handler/tagging.go#L55), [this](https://github.com/nspcc-dev/neofs-s3-gw/blob/2b905283f65a330714e38955b699a1d2abeca6bb/api/handler/tagging.go#L109) and [this](https://github.com/nspcc-dev/neofs-s3-gw/blob/2b905283f65a330714e38955b699a1d2abeca6bb/api/handler/tagging.go#L145)  because of [that](https://github.com/nspcc-dev/neofs-s3-gw/blob/2b905283f65a330714e38955b699a1d2abeca6bb/api/layer/tagging.go#L182)

File: api/handler/put.go [hunk 1]
```diff
@@ -745,7 +742,7 @@ func (h *handler) CreateBucketHandler(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	h.log.Info("bucket is created", zap.Stringer("container_id", bktInfo.CID))
+	h.log.Info("bucket is created", zap.String("reqId", reqInfo.RequestID), zap.Stringer("container_id", bktInfo.CID))
```
Comment (KirillovDenis) [inline api/handler/put.go:745]: Let's add log bucket name also. And place this between line 731 and 733. It allows to see created container even if we couldn't enable versioning.

**Review by alexvanin — APPROVED** [review]

File: api/handler/put.go (hunk 1 above)
Comment (KirillovDenis) [reply api/handler/put.go]: > And place this between line 731 and 733. It allows us to see created container even if we couldn't enable versioning.

What about this ?

Comment (ir0nbee) [reply api/handler/put.go]: Somehow  missed it. Not the logging call is in the place.

**Review by KirillovDenis — APPROVED** [review]

**Pull request merged**